src/java.base/share/classes/java/util/zip/Deflater.java

```diff
@@ -868,23 +886,42 @@ public void reset() {
     }
 
     /**
-     * Closes the compressor and discards any unprocessed input.
+     * Closes and releases the resources held by this {@code Deflater}
+     * and discards any unprocessed input.
+     * <p>
+     * If this method is invoked multiple times, the second and subsequent calls do nothing.
```

Comment:
I think this could be clearer if you replace this sentence with "If the Deflater is already closed then invoking this method has no effect."

Reply:
Done.